templates/gocd-server-homego-pvc.yml

```diff
@@ -0,0 +1,23 @@
+{{- if and .Values.server.persistence.homego.enabled (not .Values.server.persistence.homego.existingClaim) -}}
```

Comment:
Don't think this should be checked for both. If enabled, by default provide both home/go and godata

Reply:
How about enabled param being under persistence?

so, in `values.yml`, it would be

```
persistence:
    enabled: true
    godata:
      name: "godata-pvc"
    homego:
      name: "homego-pvc"
```

Reply:
Yes, works.